tests/php/Extension/Engine/SiteTreePublishingEngineTest.yml

```diff
@@ -22,3 +22,7 @@ SilverStripe\CMS\Model\SiteTree:
     Title: Page 6
     URLSegment: page-6
     ParentID: =>SilverStripe\CMS\Model\SiteTree.page5
+
+
+SilverStripe\StaticPublishQueue\Dev\DataObjectNoTrigger:
+  dataobject1:
```

Comment:
If there's no data in it, there's no need to set up a fixture. Just `::create()->write()` in the test.